web/ui/dashboard/src/app/components/modal/modal.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="fixed bg-black bg-opacity-50 h-screen w-screen top-0 right-0 bottom-0"></div>
+<div class="fixed w-full bg-white-100 h-screen top-0 right-0 bottom-0 overflow-y-auto translate-x-0 shadow" [class]="modalType === 'side' ? 'max-w-[600px]' : 'w-screen'">
+	<div class="px-20px pt-20px pb-16px border-b border-b-grey-10" *ngIf="hasModalHead">
+		<ng-content select="[modalHead]"></ng-content>
```

Comment:
Thinking about this again, we have specific head styles, so we shouldn't be passing custom content for heading. It should just be the head title, the style will be specific, e.g. modal head will always have a close button and a title, e.t.c. Do you get it?

Reply:
hmm, yes, that's better